### PR TITLE
Moving core20 snap to $devel/$nextlts series

### PR DESCRIPTION
**Pull request by xnox**
Build on eoan distro-series locally with $ sudo snapcraft. And fix things.

- Makefile
  - Try to use $devel daily tarball, until we hit $nextlts.
  - Use $nextlts once we do.
- snapcraft.yaml:
  - make version a string
  - declare bogus core18 base (core20 or base don't work)
  - add distro-info build-dep
- rename most includes from `/lib` to `/usr/lib` as we now have merged-usr
- add kvm group 112 *not sure if this was not used before for anything else*
- clean up private dir, newly created by tmpfiles

At the end of the build snapcraft blows up. So don't merge this yet.

### CODE REVIEW

File: snapcraft.yaml [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+version: "20"
 summary: Runtime environment based on Ubuntu 20.04
 description: |
   The base snap based on the Ubuntu 20.04 release.
 confinement: strict
 type: base
+## Self-reference much?! =)
+## The plugin used by part 'boostrap' does not support snaps using base 'core20'.
+#base: core20
+base: core18
```
Comment (mvo5) [inline snapcraft.yaml]: Build-base maybe if thats supported yet by snapcraft?

File: hooks/000-provide-uids-gids.chroot [hunk 2]
```diff
@@ -267,6 +268,7 @@ systemd-timesync:!::
 systemd-network:!::
 systemd-resolve:!::
 systemd-bus-proxy:!::
+kvm:!::
```
Comment (mvo5) [inline hooks/000-provide-uids-gids.chroot]: Why do we need the kvm user?

Comment (xnox) [reply hooks/000-provide-uids-gids.chroot]: See the commit message....

```
Add kvm system group  …
udev postinst adds system group kvm to ensure that /dev/kvm is owned
by the right group. There is no pre-allocated fixed groupid for
it.

Adding a new allocation kvm 112.

However, if that allocation was used in core16 before a different
(higher) one should probably be used.
```

Comment (xnox) [reply hooks/000-provide-uids-gids.chroot]: and it's not user, but group.

**Comment by xnox** [comment]
@cmatsuoka @mvo5 this is now ready to be merged. Lots of test snaps got build in https://launchpad.net/~xnox/+snap/core20 and tested by cmatsuoka and everything seems to work.

This will switch core20 snap to use eoan.

**Comment by mvo5** [comment]
@xnox Looks like travis is unhappy right now, I wonder if we can fix this somehow?

**Comment by cmatsuoka** [comment]
Add `--use-lxd` (or `--destructive-mode`) to snapcraft when building the snap. Possibly `TESTDIR` in the Makefile will have to be adjusted as well.

**Comment by xnox** [comment]
@mvo5 @cmatsuoka it's now green!

**Pull request merged**